raiden/transfer/mediated_transfer/events.py

```diff
@@ -780,7 +783,8 @@ def to_dict(self) -> Dict[str, Any]:
             "amount": str(self.amount),
             "expiration": str(self.expiration),
             "secrethash": serialize_bytes(self.secrethash),
-            "signed_secret_request": self.signed_secret_request
+            "signed_secret_request": self.signed_secret_request,
+            "sender": self.sender
```

Comment:
Here we use   ` "sender": to_checksum_address(self.sender),` in order to persist the address in the same way always.